test/tests.py

```diff
@@ -544,6 +543,19 @@ def verify_dmlc_messages(self, stderr, expected_msgs):
             self.expect_messages(
                 kind, actual_msgs.get(kind, []), expected_msgs.get(kind, []))
 
+def _unittest_parse_messages():
```

Comment:
Why does it have a `_` prefix to begin with?

Reply:
I have no opinion, I can remove it if you prefer.

Reply:
Done.